backend/utils/redis/cacher.js

```diff
@@ -2,30 +2,42 @@ const croner = require("croner");
 const {
   getAllProposalRecords,
   getCurrentRoundProposals,
+  getProposalByRecordId,
 } = require("../airtable/utils");
 const redis = require("./index");
 
 async function cacheProposal(recordId, fields) {
   await redis.json.set(recordId, ".", fields);
 }
 
+function _deleteReduntantFields(proposal) {
+  // remove fields that are not needed
+  delete proposal.fields["Grant Deliverables"];
+  delete proposal.fields["Deliverable Checklist"];
+  delete proposal.fields["Proposal Title"];
+  delete proposal.fields["Wallet Address"];
+  delete proposal.fields["One Liner"];
```

Comment:
aren't this needed on update?

Reply:
yep, it aren't

Reply:
We already have these fields in mongo, to avoid duplication and save space